tools/scylla-nodetool.cc

```diff
@@ -735,6 +739,8 @@ command-line arguments, the compaction will run on these tables.
 Fore more information, see: https://opensource.docs.scylladb.com/stable/operating-scylla/nodetool-commands/compact.html
 )",
                 {
+                    typed_option<>("skip-flush,sf", "skip flushing tables before major compaction"),
```

Comment:
Note that boost doesn't support multi-char short-form parameter names. I think this will have to be `--sf` on the command line. I think it is better to have a proper short form (invokable with `-`) or no short form at all.

Reply:
Missing: update test in `test/nodetool`.

Reply:
added tests in `test/nodetool/test_compact.py` in next version

Reply:
option supports only `"skip-flush"` in next version